### DISCUSSION TITLE
Allow transmog of Main Hands and Off Hands on One-Handed Weapons

### DISCUSSION THREAD

**Discussion by ishidinho** [chromiecraft/chromiecraft#4926] [Suggestions]
The latest setting changes to transmog are great! What we are now missing is the ability to transmog Main Hand and Off Hand weapons to One-Handed Weapons and vice versa.

With the start of TBC, Main Hand and Off Hand weapons are pretty rare, because most of the weapons are just one-handed. I think people would love to use their Quel'Serrars or Linken's Sword again.

**Comment by Gultask** [comment]
The initial idea was to implement a system more closely resembling the current modern set-up found in Retail WoW, with more liberties. This system still has MH/OH restrictions. However, we've noticed 2 things after implementing this system:

1. Rogues are much more affected by this change than other classes. Not only because they dual-wield a lot. But also because daggers are exclusive and can't be transmogged into other weapon types, unlike axes and staves.
2. In Retail, this restriction is moot since there's virtually no Main Hand or Off Hand items anymore. They're all either Two-Handed or One-Handed.

So I asked the contributor that made that first rework, ZhengPeiRu, to also add this Handedness restrictions as a configurable setting. And she did so here: https://github.com/azerothcore/mod-transmog/pull/107

So now with the next update we can toggle that option on or off on the server's side :)
